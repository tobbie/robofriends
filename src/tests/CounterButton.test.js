import {shallow} from 'enzyme'
import React from 'react'
import CounterButton from '../components/CounterButton'

describe('counter button tests',()=> {
    let wrapper;
    

    it('expect to render counter button component',() => {
        wrapper = shallow(<CounterButton />)
        expect(wrapper).toMatchSnapshot();
        
        
    });

    it('expect to render counter to increment',() => {
        wrapper = shallow(<CounterButton />)
        wrapper.find('[id="ctrButton"]').simulate('click');
        expect(wrapper.state()).toEqual({count: 1})
        wrapper.find('[id="ctrButton"]').simulate('click');
        expect(wrapper.state()).toEqual({count:2})
        
        
    });




})