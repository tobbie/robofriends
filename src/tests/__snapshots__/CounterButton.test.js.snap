// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counter button tests expect to render counter button component 1`] = `ShallowWrapper {}`;
